website/app/layout.tsx

```diff
@@ -35,12 +35,8 @@ const RootLayout: FC<{
           'https://github.com/the-guild-org/the-guild-components/tree/main/website',
       }}
       navbarProps={{
-        navLinks: [
-          {
-            href: '/docs',
-            children: 'Documentation',
-          },
-        ],
+        developerMenu: [],
```

Comment:
We'll have a conflicts. I just added a menu to v8-3.

Reply:
I needed it, otherwise, I had a runtime error